### ISSUE TITLE
Adding GPG

### ISSUE THREAD

**Issue by Eric-Sparks** [candlepin/candlepinproject.org#94]
It would be great if the training could address GPG.

**Comment by awood** [comment]
@Eric-Sparks Good idea.  The talk is already 2.5 hours so I might make that a supplementary section.  Also I'll need to learn a little bit more about gpg :smile: (Enigmail does all the heavy lifting for me).
